core/benches/banking_stage.rs

```diff
@@ -167,9 +167,6 @@ fn bench_banking(bencher: &mut Bencher, tx_type: TransactionType) {
     let (verified_sender, verified_receiver) = unbounded();
     let (tpu_vote_sender, tpu_vote_receiver) = unbounded();
     let (vote_sender, vote_receiver) = unbounded();
-    let mut bank = Bank::new_for_benches(&genesis_config);
-    // Allow arbitrary transaction processing time for the purposes of this bench
-    bank.ns_per_slot = std::u128::MAX;
```

Comment:
I would agree that the update to `ns_per_slot` is currently getting discarded; however, I believe this was probably unintentional. So, I think we should correct the logic to make the update actually occur instead of ripping it out:

```
    let mut bank = Bank::new_for_benches(&genesis_config);
    // Allow arbitrary transaction processing time for the purposes of this bench
    bank.ns_per_slot = std::u128::MAX;
    let bank = Arc::new(bank);
```

Reply:
Alright can do.